src/dapp-tests/pass/cheatCodes.sol

```diff
@@ -87,4 +102,54 @@ contract CheatCodes is DSTest {
         (string memory output) = abi.decode(hevm.ffi(inputs), (string));
         assertEq(output, "acab");
     }
+
+    function testNonce(address who, uint n) public {
+        hevm.file("nonce", who, n);
+        assertEq(hevm.look("nonce", who), n);
+    }
+
+    function testBalance(address who, uint bal) public {
+        hevm.file("balance", who, bal);
+        assertEq(who.balance, bal);
+    }
+
+    function testReplace() public {
```

Comment:
no idea why at all, but if I construct the target contract directly in the test instead of in the constructor this test always fails (and it does so before reaching the call to the cheatcode).... :thinking: 

```suggestion
    function testReplace() public {
        Old target = new Old();
```

Reply:
Hmm, strange indeed. Does it revert or what is the error?

Reply:
```
Failure: testReplace
  src/env.sol:CheatCodes
   ├╴constructor
   │  └╴create <unknown contract>@0xCe71065D4017F316EC606Fe4422e11eB2c47c246 <no source map>
   │     └╴← 63 bytes of code
   └╴testReplace()
      └╴error Revert 0x <no source map>
  ```